app/styles/app/modules/pagination.scss

```diff
@@ -5,7 +5,7 @@
   ul {
     display: flex;
     align-items: baseline;
-    flex-wrap: wrap;
+    flex-wrap: nowrap;
```

Comment:
Hmm yeah the way it was wrapping before didn't look great, but doesn't this cause the outer edges of the pagination to go offscreen when there are a lot of items on a narrow screen? For example https://as-pagination-refactoring.test-deployments.travis-ci.org/travis-ci

Reply:
Hmm, you're right 🤔But I think it should rather collapse than wrap.. I'll think about that.

Reply:
It appears that the best way would be to change `inner` and `outer` parameters depending on window dimensions. But this seems like a significant change, so I'll just revert it to wrap for now and file a separate issue.

Reply:
Cool that sounds good to me. I agree that the current wrapping isn't great and some sort of collapse sounds like a good solution. But yeah that sounds like a significant change, so moving it into a separate issue sounds good 👍 Thanks!